helmfile/charts/prometheus-alerts/values.yaml

```diff
@@ -7,6 +7,7 @@ alertmanagerNamespace: monitoring
 prometheusJob: kube-prometheus-stack-prometheus
 operatorJob: kube-prometheus-stack-operator
 prometheusNamespace: monitoring
+predictLinearLimit: 0.66
```

Comment:
Can we have this as a config option as well?

Reply:
Done!